census_rm_ops_ui/templates/view_case_details.html

```diff
@@ -0,0 +1,21 @@
+{% extends "layouts/base.html" %}
```

Comment:
We've used 2 spaces indents on the other HTML templates, would be nice to format this one to match 💅